core/src/main/java/org/mule/config/DefaultMuleConfiguration.java

```diff
@@ -204,6 +204,11 @@
      */
     private ProcessingStrategy defaultProcessingStrategy;
 
+    /**
+     * {@inheritDoc}
```

Comment:
I'd typically expect this on getter though, not on private attribute, but this is minor.